.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: Feature/LAZ-
+labels: ''
+assignees: ''
+
+---
+
+**JIRA Ticket**
+https://perigongames.atlassian.net/browse/LAZ- *[TICKET NUMBER]*
```

Comment:
Looks good,

can you also add a gif/img section

Reply:
done